pkg/multicluster/local/local_cluster_manager.go

```diff
@@ -8,49 +8,42 @@ import (
 	"fmt"
 
 	"emperror.dev/errors"
-	"fybrik.io/fybrik/manager/apis/app/v1alpha1"
-	"fybrik.io/fybrik/pkg/multicluster"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"fybrik.io/fybrik/manager/apis/app/v1alpha1"
+	"fybrik.io/fybrik/pkg/multicluster"
```

Comment:
Why is the order of imports changing? Shouldn't it be sorted lexicographically?

Reply:
In most of OpenSource projects that I've seen, there is an imports separation between standard golang libraries, third parties and imports from the same repository.
For example, her is a file from  kubernetes-sigs /kubefed  https://github.com/kubernetes-sigs/kubefed/blob/master/pkg/controller/schedulingmanager/controller.go.